integration_tests/lpython_decorator_02.py

```diff
@@ -0,0 +1,15 @@
+from numpy import array
+from lpython import i32, f64, lpython
+
+@lpython
+def multiply(n: i32, x: f64[:]) -> f64[:]:
```

Comment:
This needs to be changed to:
```python
n = TypeVar("n")
def multiply(n: i32, x: f64[:]) -> f64[n]:
```
The return array must have a length that is determined using the function arguments.